### PR TITLE
[HOPSWORKS-2543] add support to insert_stream() to ingest data into offline fs

### PR DESCRIPTION
**Pull request by davitbzh**

### CODE REVIEW

File: recipes/yarn.rb [hunk 1]
```diff
@@ -198,15 +215,15 @@
     paths [
             "#{Chef::Config['file_cache_path']}/#{hopsExamplesSpark}", 
             "#{Chef::Config['file_cache_path']}/#{hopsExamplesFeaturestoreTour}", 
-            "#{Chef::Config['file_cache_path']}/#{hsfs_utils}",
+            "#{Chef::Config['file_cache_path']}/#{hsfs_utils_py}",
```
Comment (SirOibaf) [inline recipes/yarn.rb:218]: Here we need to do the same also for the `hsfs_util_java` otherwise it won't work on hopsworks.ai

**Review by SirOibaf — CHANGES_REQUESTED** [review]

File: attributes/default.rb [hunk 2]
```diff
@@ -120,12 +120,14 @@
...
 default['hadoop_spark']['hsfs']['utils']['version']         = node['install']['version']
-default['hadoop_spark']['hsfs']['utils']['download_url']    = "#{node['download_url']}/hsfs_utils/hsfs_utils-#{node['hadoop_spark']['hsfs']['utils']['version']}.py"
+default['hadoop_spark']['hsfs']['utils']['py_download_url']    = "#{node['download_url']}/hsfs_utils/hsfs_utils-#{node['hadoop_spark']['hsfs']['utils']['version']}.py"
+default['hadoop_spark']['hsfs']['utils']['java_download_url']    = "#{node['download_url']}/hsfs_utils/hsfs_utils-#{node['hadoop_spark']['hsfs']['utils']['version']}.jar"
 
 #
 # Hudi Dependencies
 #
 default['hadoop_spark']['hudi_bundle_url']                  = "#{node['download_url']}/hudi/#{node['hive2']['hudi_version']}/hudi-spark3-bundle_#{node['scala']['version']}-#{node['hive2']['hudi_version']}.jar"
+default['hadoop_spark']['hudi_bundle_url']                  = "#{node['download_url']}/hudi/#{node['hive2']['hudi_version']}/hudi-utilities-bundle_#{node['scala']['version']}-#{node['hive2']['hudi_version']}.jar"
```
Comment (moritzmeister) [inline attributes/default.rb:130]: Aren't you overwriting the attribute here again?

Comment (davitbzh) [reply attributes/default.rb:130]: https://github.com/logicalclocks/spark-chef/pull/142

File: attributes/default.rb [hunk 3]
```diff
@@ -120,7 +120,8 @@
 default['hadoop_spark']['hsfs']['url']                      = "#{node['download_url']}/hsfs/#{node['hadoop_spark']['hsfs']['version']}/hsfs-#{node['hadoop_spark']['hsfs']['version']}.jar"
 
 default['hadoop_spark']['hsfs']['utils']['version']         = node['install']['version']
-default['hadoop_spark']['hsfs']['utils']['download_url']    = "#{node['download_url']}/hsfs_utils/hsfs_utils-#{node['hadoop_spark']['hsfs']['utils']['version']}.py"
+default['hadoop_spark']['hsfs']['utils']['py_download_url']    = "#{node['download_url']}/hsfs_utils/hsfs_utils-#{node['hadoop_spark']['hsfs']['utils']['version']}.py"
+default['hadoop_spark']['hsfs']['utils']['java_download_url']    = "#{node['download_url']}/hsfs_utils/hsfs_utils-#{node['hadoop_spark']['hsfs']['utils']['version']}.jar"
```
Comment (moritzmeister) [inline attributes/default.rb:124]: In the jar name it should be a dash not underscore

**Review by moritzmeister — APPROVED** [review]

**Pull request merged**